<launch>
  <include file="$(find pr2_alpha)/$(env ROBOT).machine" />


<!-- Actions -->
  <node pkg="annotated_map_builder" type="wait_for_k_messages_action.py" machine="four" name="wait_k_messages_action" output="screen">
    <remap from="topic" to="/stereo/raw_stereo_throttled"/>
    <param name="count" value="3" />
  </node>


  <include file="$(find annotated_map_builder)/test/move_head.launch" />
  <include file="$(find annotated_map_builder)/test/move_head2.launch" />






</launch>
